packages/ra-ui-materialui/src/form/TabbedForm.spec.tsx

```diff
@@ -46,7 +46,8 @@ describe('<TabbedForm />', () => {
 
         const tabs = await screen.findAllByRole('tab');
         expect(tabs.length).toEqual(2);
-        await screen.findByLabelText('Title');
+        const titleInput = await screen.findByLabelText('Title');
+        expect(titleInput).toBeVisible();
```

Comment:
Had to add this extra check because the current test did not detect the regression :grimacing: